build/Dockerfile

```diff
@@ -20,7 +20,7 @@ FROM ${BASE_IMAGE}
 LABEL name="StorageOS Cluster Operator" \
     maintainer="support@storageos.com" \
     vendor="StorageOS" \
-    version="v2.2.1" \
+    version="v2.3.0" \
     release="1" \
```

Comment:
Just noticed `release="1"`.  We should probably upldate to 2?  Maybe for next release.